modules/persistent-storage-csi-secrets-store-network-policies.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-secrets-store.adoc
+//
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-secrets-store-network-policies_{context}"]
+= Support for network policies
+
+The {secrets-store-operator} includes pre-defined `NetworkPolicies` resources for enhanced security. These policies govern the ingress and egress traffic for both the SS-CSI Operator and its associated driver.
+
+The following table summarizes the default ingress and egress rules:
+
+[cols="1,1,1,1", options="header"]
+|===
+| Component | Ingress ports | Egress ports | Description
+
+| {secrets-store-operator}
+| 8443
+| 6443
+| Accesses metrics and communicates with the API server
+
+| {secrets-store-driver}
+| 8095
+| 6443
```

Comment:
```suggestion
| {secrets-store-operator}
| `8443`
| `6443`
| Accesses metrics and communicates with the API server

| {secrets-store-driver}
| `8095`
| `6443`
```


Consider using back ticks for port number/values
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference